workflows/nimble_workflows.py

```diff
@@ -0,0 +1,311 @@
+import sys
```

Comment:
Should we put these in separate files?  It might be easier for a user to navigate and find the workflow of interest.

Reply:
agree